src/main/java/labs/introtoprogramming/lab5/scene/Transform.java

```diff
@@ -0,0 +1,47 @@
+package labs.introtoprogramming.lab5.scene;
+
+import labs.introtoprogramming.lab5.geometry.Vector3;
+
+public class Transform {
+  private Vector3 position;
+  private Vector3 rotation;
+  private Vector3 scale;
+
+  public Transform(Vector3 position, Vector3 rotation) {
+    this.position = position;
+    this.rotation = rotation;
+    this.scale = Vector3.one();
+  }
+
+  public Transform(Vector3 position) {
+    this(position, Vector3.zero());
+  }
+
+  public Transform() {
+    this(Vector3.zero(), Vector3.zero());
+  }
+
+  public Vector3 position() {
+    return position.multiply(1);
```

Comment:
Why not return position? Or just clone it?

Reply:
I create copy of vector by multiplying it by one or is there better way?

Reply:
Better implement Cloneable and use `clone()`